static/index.html

```diff
@@ -6,28 +6,29 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 
-		<title>CF-Deck</title>
+		<title>18F Cloud Deck</title>
 
-		 <!-- Bootstrap -->
-    	<link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+		<!-- Bootstrap -->
+		<link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+		<link href="assets/css/app.css" rel="stylesheet">
 
 		<script src="bower_components/angularjs/angular.js"></script>
 		<script src="bower_components/angular-route/angular-route.js"></script>
-		<script src="js/app/main.js"></script>
-		<script src="js/app/cloudfoundry.js"></script>
-		<script src="js/app/controllers.js"></script>
-		<script src="js/app/filters.js"></script>
+		<script src="app/main.js"></script>
+		<script src="app/cloudfoundry.js"></script>
+		<script src="app/services.js"></script>
+		<script src="app/controllers.js"></script>
+		<script src="app/filters.js"></script>
 	</head>
-	<body>
-		<div class="site-wrapper">
-			<div class="site-wrapper-inner">
-				<div class="container">
-				<!-- Where our view will load -->
-	    		<div ng-view></div>
+	<body ng-controller="MainCtrl">
```

Comment:
I think that we shouldn't allow any views without being logged in. We should redirect from the go app to the login page.